ingredient/views.py

```diff
@@ -1,3 +1,27 @@
-from django.shortcuts import render
+from django.core.exceptions import ObjectDoesNotExist
+from django.http            import JsonResponse
+from django.views           import View
+from ingredient.models      import Ingredient
 
-# Create your views here.
+
+class IngredientView(View):
+    def get (self, request, ingredient_id): 
+        try: 
+            ingredient  = Ingredient.objects.get(id=ingredient_id)
+            recipe_all  = ingredient.recipe.all()
+            recipe_list = [{
+                "id"        : recipe.id,
+                "name"      : recipe.name,
+                "image_url" : recipe.image_url} for recipe in recipe_all]
+            result = {
+                "id"               : ingredient.id,
+                "name"             : ingredient.name,
+                "price"            : ingredient.price,
+                "info"             : ingredient.info,
+                "storage"          : ingredient.storage,
+                "image_url"        : ingredient.image_url,
+                "related_recipe"   : recipe_list
+            }
```

Comment:
한줄이라도 더 줄여보면 어떨까요? 그리고 `related_recipe` 보다는 `related_recipes`가 좋을 것 같습니다! 

```suggestion
            ingredient  = Ingredient.objects.get(id=ingredient_id)
            result      = {
                "id"               : ingredient.id,
                "name"             : ingredient.name,
                "price"            : ingredient.price,
                "info"             : ingredient.info,
                "storage"          : ingredient.storage,
                "image_url"        : ingredient.image_url,
                "related_recipes"  : [{
                    "id"        : recipe.id,
                    "name"      : recipe.name,
                    "image_url" : recipe.image_url
                } for recipe in ingredient.recipe.all()]
            }
```

Reply:
이렇게도 코드 줄일 수 있군요! 감사합니다~